setup.py

```diff
@@ -5,10 +5,7 @@
     long_description = fh.read()
 
 setup(
-    install_requires=[
-        "requests",
-        "camel-converter[pydantic]"
-    ],
+    install_requires=["requests", "camel-converter[pydantic]", "black", "isort"],
```

Comment:
Black and isort are development dependencies so not needed here. The `pipenv install --dev <package-name>` @alallema gave will add them to the needed files. 

```suggestion
    install_requires=["requests", "camel-converter[pydantic]"],
```

Reply:
Yes, it's a good thing you check better than I do 😄